docs/upgrade/UPGRADE-v7.1.0.md

```diff
@@ -0,0 +1,173 @@
+        ```diff
+        snc_redis:
+            clients:
+        +       global:
+        +           type: 'phpredis'
+        +           alias: 'global'
+        +           dsn: 'redis://%redis_host%'
+        +           options:
+        +               prefix: '%env(REDIS_PREFIX)%'
+        ```
+    - to `src/Shopsys/ShopBundle/Resources/config/services/commands.yml` add
+        ```
+        Shopsys\FrameworkBundle\Command\RedisCleanCacheOldCommand: ~
+        ```
+    - create `app/config/parameters_version.yml.dist` with following content
+        ```yml
+        parameters:
+            build-version: %%version%%
+        ```
+    - to `build.xml` add new phing targets
+        ```xml
+        <target name="clean-redis-old" description="Cleans up redis cache for previous build versions">
+            <exec executable="${path.php.executable}" passthru="true" checkreturn="true" output="${dev.null}">
+                <arg value="${path.bin-console}" />
+                <arg value="shopsys:redis:clean-cache-old" />
+            </exec>
+        </target>
+        <target name="generate-build-version">
+            <exec executable="${path.php.executable}" checkreturn="true" outputProperty="version">
+                <arg value="-r" />
+                <arg value="echo date('YmdHis');" />
+            </exec>
+            <copy file="${path.app}/config/parameters_version.yml.dist" tofile="${path.app}/config/parameters_version.yml" overwrite="true">
+                <filterchain>
+                    <replacetokens begintoken="%%" endtoken="%%">
+                        <token key="version" value="${version}" />
+                    </replacetokens>
+                </filterchain>
+            </copy>
+        </target>
+        ```
+    - download [`RedisVersionsFacadeTest.php`](https://github.com/shopsys/project-base/tree/master/tests/ShopBundle/Functional/Component/Redis/RedisVersionsFacadeTest.php) to your tests directory `tests/ShopBundle/Functional/Component/Redis/`
+    - run `php phing generate-build-version`
+    - and include `generate-build-version` and `clean-redis-old` to your build phing targets. Please find inspiration in [#886](https://github.com/shopsys/shopsys/pull/886/files)
+    - once you finish this change (include the `build-version` into caches), you still should deal with older redis cache keys that don't use `build-version` prefix (16 digits).
+      Such keys are not removed even by `clean-redis-old`, please find and remove them manually (via console or UI)
+- *(low priority)* remove option `choice_name` from `flags` and `brands` in `ShopBundle/Form/Front/Product/ProductFilterFormType.php` ([#891](https://github.com/shopsys/shopsys/pull/891))
+- create `Shopsys\ShopBundle\Model\AdvancedSearch\ProductAdvancedSearchConfig` by extending `Shopsys\FrameworkBundle\Model\AdvancedSearch\ProductAdvancedSearchConfig` ([#895](https://github.com/shopsys/shopsys/pull/895))
```

Comment:
hm, no *low priority* here, however, this should have been resolved at the time when the particular PR was merged so I suppose it is ok. I will raise the topic on our team meeting.